javascript.js

```diff
@@ -0,0 +1,105 @@
+const dropdown = document.querySelector(".dropdown");
+const dropdownList = document.querySelector(".dropdown__list");
+const dropdownBefore = window.getComputedStyle(dropdown,"::before");
+const droplink = document.querySelector(".nav__link--dropdown");
+const contact = document.querySelector(".nav__item--contact");
+const navbar = document.querySelector(".navbar");
+const burgerBtn = document.querySelector(".navbar__hamburger");
+const hero = document.querySelector(".hero");
+const sliderBtnActive = document.querySelectorAll(".hero__sliderBtn");
+
+
+let menuOpen = false;
+let dropdownOpen = false;
+
+burgerBtn.addEventListener("click",()=>{
+	if (!menuOpen){
+		burgerBtn.classList.remove("closed");

```

Comment:
e corect si closed, dar ar fi ok sa te obisnuiesti cu is-closed, is-active, etc. sunt ca niste state-uri